whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceAclTestIntegration.java

```diff
@@ -65,6 +67,50 @@ class RdapServiceAclTestIntegration extends AbstractRdapIntegrationTest {
             "last-modified:  2001-02-04T17:00:00Z\n" +
             "source:    TEST\n");
 
+    private static final RpslObject TEST_INETNUM = RpslObject.parse("" +
+                    "inetnum:        0.0.0.0 - 255.255.255.255\n" +
+                    "netname:        IANA-BLK\n" +
+                    "descr:          The whole IPv4 address space\n" +
+                    "country:        NL\n" +
+                    "tech-c:         TP1-TEST\n" +
+                    "admin-c:        TP1-TEST\n" +
+                    "status:         OTHER\n" +
+                    "mnt-by:         OWNER-MNT\n" +
+                    "created:         2022-08-14T11:48:28Z\n" +
+                    "last-modified:   2022-10-25T12:22:39Z\n" +
+                    "source:         TEST");
+
+    private static final RpslObject TEST_INE6TNUM = RpslObject.parse("" +
+                    "inet6num:       ::/0\n" +
+                    "netname:        IANA-BLK\n" +
+                    "descr:          The whole IPv6 address space\n" +
+                    "country:        NL\n" +
+                    "tech-c:         TP1-TEST\n" +
+                    "admin-c:        TP1-TEST\n" +
+                    "status:         OTHER\n" +
+                    "mnt-by:         OWNER-MNT\n" +
+                    "created:         2022-08-14T11:48:28Z\n" +
+                    "last-modified:   2022-10-25T12:22:39Z\n" +
+                    "source:         TEST");
+    private static final RpslObject TEST_ROLE = RpslObject.parse("" +
```

Comment:
Also test that a role entity e-mail is returned and ACL accounted

Reply:
Created a new test for the ROLE